src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_RADIUS_OF_CIRCLE = 10;
+    private static final String DEFAULT_NAME_OF_CIRCLE = "CIRCLE";
+    private static final String DEFAULT_COLOR_OF_CIRCLE = "WHITE";
+    private static final int MAX_RANDOM_VALUE = 20;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private String randomColor;
+    private String figureName;

```

Comment:
you can create this variables inside method getRandomFigure(), you use it only inside this method and don't need to initialize it when you create FigureSupplier object